web/packages/teleport/src/Roles/RoleEditor/StandardEditor/standardmodel.test.ts

```diff
@@ -1101,7 +1096,7 @@ describe('roleEditorModelToRole', () => {
   it('converts metadata', () => {
     expect(
       roleEditorModelToRole({
-        ...minimalRoleModel(),
+        ...minimalRoleModel(RoleVersion.V5),
```

Comment:
Why v5 here?

Reply:
This test case was based on role v5, see [line 1117](https://github.com/gravitational/teleport/pull/54738/files/7076a01ffa049d05a095ff3c9d126e74431859f7#diff-343ae72e01f166e90f26e00bc5232fb7015f36955883902899af06d07146cb3aR1117) so I only kept the version as is. I could have used v7 though!